cli/src/main.rs

```diff
@@ -84,6 +86,22 @@ fn build_command() -> Command<'static> {
                 .takes_value(true)
                 .required(false),
         )
+        .arg(
+            Arg::new(ARG_SCALA_PACKAGE)
+                .short('S')
+                .long("scala-package")
+                .help("Scala package name")
+                .takes_value(true)
+                .required(false),
+        )
+        .arg(
+            Arg::new(ARG_SCALA_MODULE_NAME)
+                .short('M')
```

Comment:
Same issue here, there's no clear reason why uppercase would mean 'Scala module name' and lowercase would mean 'Kotlin module name'. I propose that we get rid of both short args, although we should probably hold off on changing the Kotlin args right now since that would be a breaking change.